### PR TITLE
[exporter/kafka] Support sending individual log records

### PR DESCRIPTION
**Pull request by nanorobocop**
**Description:** Support sending individual log records

**Link to tracking Issue:** https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/12623

**Testing:** Tested locally

**Documentation:** Updated README, Changelog

### CODE REVIEW

**Comment by djaglowski** [comment]
Pinging code owners: @pavolloffay @MovieStoreGuy

File: exporter/kafkaexporter/log_record_marshaler.go
Comment (pavolloffay) [inline exporter/kafkaexporter/log_record_marshaler.go:69]: should be this TODO handled before merging?

Comment (nanorobocop) [reply exporter/kafkaexporter/log_record_marshaler.go:69]: I actually thought this could be done in attribute processor, so not much needed here.

**Review by tigrannajaryan — CHANGES_REQUESTED** [review]
I already [commented](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/12623#issuecomment-1199814693) in the linked issue and don't see my comment addressed. Copying it here:

>One possible option would be to add support for other encodings to kafka exporter. **However, the encoding needs to be well defined and we need to have strong arguments in favour of adding such encoding to Kafka exporter.**

Blocking this until there is a well-defined description of the format with arguments in favour of the proposed format with consideration of alternate formats. This likely requires an OTEP that proposes a flat JSON representation for OTLP.

**Comment by MovieStoreGuy** [comment]
Sorry I am late coming to this, but I agree with @tigrannajaryan

**Pull request closed without merging**
